Sep3DataTier/Repository/DatabaseContext.cs

```diff
@@ -34,9 +34,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.Entity<Location>().HasKey(location => location.Id);
         modelBuilder.Entity<Model.Report>().HasKey(report => report.Id);
         modelBuilder.Entity<ApplicationUser>().HasKey(user => user.Id);
-        modelBuilder.Entity<ApplicationUser>().HasIndex(u => u.Email).IsUnique();
-        modelBuilder.Entity<IdentityRole>().HasData(GenerateIdentityRole("User"));
-        modelBuilder.Entity<IdentityRole>().HasData(GenerateIdentityRole("Admin"));
+        modelBuilder.Entity<ApplicationUser>().HasIndex(user => user.Email).IsUnique();
+        // modelBuilder.Entity<IdentityRole>().HasData(GenerateIdentityRole("User"));
+        // modelBuilder.Entity<IdentityRole>().HasData(GenerateIdentityRole("Admin"));
```

Comment:
Is there a reason for commenting that?

Reply:
For some reason, migrations weren't working for me when it wasn't commented out.